web/src/pages/Route/transform.ts

```diff
@@ -35,8 +44,14 @@ export const transformStepData = ({
     };
   }
 
+  const labels = {};
+  transformLableValueToKeyValue(form1Data.labels).forEach(item => {
+    labels[item.labelKey] = item.labelValue;
```

Comment:
if we have `version:v2` and `version:v3`, only the laster one will be posted, need to confirm. 
cc @juzhiyuan @nic-chen

Reply:
both of them need to be posted